toxics/slicer.go

```diff
@@ -37,10 +37,13 @@ func (t *SlicerToxic) chunk(start int, end int) []int {
 		return []int{start, end}
 	}
 
+	mid := start + (end-start)/2
 	// +1 in the size variation to offset favoring of smaller
 	// numbers by integer division
 	//#nosec
```

Comment:
The comment is still valid. `rand.Intn(2)` is the correct call for creating a uniform random over {0, 1}. The comment is helpful because otherwise I could not have guessed what the call was for.